### PR TITLE
Update contributing documentation

### PR DESCRIPTION
**Pull request by AlekSi**
* added changes for draft PRs;
* merged main and team PR templates as we never used the latter;
* added `task godocs` target.

### CODE REVIEW

**Review by seeforschauer — DISMISSED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -18,3 +18,7 @@ This PR closes #{issue_number}.
 
 * [ ] I added tests for new functionality or bugfixes.
 * [ ] I ran `task all`, and it passed.
+* [ ] I added/updated comments for both exported and unexported top-level declarations.
+* [ ] I checked comments rendering with `task godocs`.
+* [ ] (for maintainers only) I set Reviewers, Assignee, Labels, Project and project's Sprint fields.
+* [ ] I checked all relevant checklist items.
```
Comment (rumyantseva) [inline .github/PULL_REQUEST_TEMPLATE.md:24]: This doesn't look really straightforward. Is it about this checklist or checklist in the relevant issue? Let's specify.

**Review by rumyantseva — COMMENTED** [review]
One tiny question about the last item of the checklist.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -18,3 +18,7 @@ This PR closes #{issue_number}.
 
 * [ ] I added tests for new functionality or bugfixes.
 * [ ] I ran `task all`, and it passed.
+* [ ] I added/updated comments for both exported and unexported top-level declarations.
```
Comment (w84thesun) [inline .github/PULL_REQUEST_TEMPLATE.md:21]: I suggest adding an explanation here too, as `top-level declarations` is a bit confusing from the first read.
```suggestion
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
```

**Review by w84thesun — CHANGES_REQUESTED** [review]
LGTM, except that small change.

**Review by rumyantseva — APPROVED** [review]
LGTM

**Review by w84thesun — APPROVED** [review]

**Review by seeforschauer — APPROVED** [review]

**Pull request merged**